documentation/MeetingMinutes/2024/MOM-2024-02-14.md

```diff
@@ -0,0 +1,134 @@
+
+#### *February 14, 2024*
+
+<br>
+
+Preliminary Minutes of Meeting. To be updated during and after the meeting.
+
+## Attendees
+
+[Gaurav Agarwal](https://wiki.camaraproject.org/display/~gaurav2192)
+
+[Herbert Damker](https://wiki.camaraproject.org/display/~hdamker)
+
+[Jesús Peña García-Oliva](https://github.com/jpengar)
+
+[Diego Gonzalez Martínez](https://github.com/diegogonmar)
+
+Sönke Peters, Vodafone
+
+[Bjorn Hjelm (OIDF)](https://github.com/bhjelm) 
+
+[Dawid Wroblewski](https://github.com/DT-DawidWroblewski)
+
+[Huub Appelboom](https://github.com/HuubAppelboom)
+
+[Tanja De Groot](https://wiki.camaraproject.org/display/~TanjaDeGroot)
+
+[Karabulut, Murat](https://wiki.camaraproject.org/display/~gmuratk)
+
+Toyeeb Rehman (GSMA)
+
+Rajesh Murthy 
+
+[Padgaonkar, Shilpa](https://wiki.camaraproject.org/display/~shilpa.padgaonkar) 
+
+[Joseph Heenan](https://github.com/jogu)
+
+[Artych, Rafał](https://wiki.camaraproject.org/display/~rart)
+
+[Surajj Jaggernath](https://wiki.camaraproject.org/display/~surajjj)
+
+[Parichaya Shrivastava](https://wiki.camaraproject.org/display/~lfsfn)
+
+[Elisabeth Mueller](https://wiki.camaraproject.org/display/~elisabethmueller)
+
+[Foo Ming Hui](https://wiki.camaraproject.org/display/~mhfoo)
+
+[Sachin Kumar](https://wiki.camaraproject.org/display/~sachinvodafone)
+
+DAVID VALLEJO NAVARRO
+
+[Diego González Martínez](https://wiki.camaraproject.org/display/~diegogonmar)
+
+[Jan Friman](https://wiki.camaraproject.org/display/~Jan_Friman)
+
+Juan Antonio Hernando La...
+
+Juan Fabio Garcia Solero
+
+PEDRO BALLESTEROS HERRANZ
+
+[Ramesh Shanmugasundaram](https://wiki.camaraproject.org/display/~sfnuser)
+
+<br>
+
+## Agenda
+
+1. Recent updates & recap
+2. Issues and PRs. Priority discussions (most active issues and/or dependencies for release v0.1):
+    - **Camara profile of OIDF and IETF standards**
+        - Please comment on https://github.com/camaraproject/IdentityAndConsentManagement/issues/122 .
+    - **offline access** #123 https://github.com/camaraproject/IdentityAndConsentManagement/pull/123
+    - **Discussion on Issue #100 "Claims to be used in oAuth Access tokens".**
+    - **Discussion on Issue #108 "Question to documentation update for security schemes".**
+    - **Discussion on Issue #104 "Alignment of Client Credential authentication / authorization using API Gateway Pattern".**
+
+3. Action Points
+4. AoB
+
+<br>
+
+## Meeting Technicallities
+
+Jesús offered that ownership of the meeting invite could be transferred to the new chair. Axel said that for now this can stay as it is.
+
+
+## Discussion on Consensus in Camara Working Groups
+
+Recapitulation of the previous discussion on this topic: Jesús and Diego said there was consensus achieved during a WG call that we continue working on https://github.com/camaraproject/IdentityAndConsentManagement/pull/113
+Herbert said that lazy-consensus is the Camara way. Shilpa said no consensus was achieved. 
```

Comment:
Why we need to document this discussion about the previous meeting at all in the minutes - it isn't an outcome which brings us forward. Objections were raised (if you want: after the meeting, based on the minutes) so the discussion had to be reopened (there wasn't a formal vote which would require a supermajority vote to reopen the decision).

Normally decision in a sub project meeting will be confirmed by conclusive actions within GitHub (e.g. approvals of a discussed PR). Here we have now the case that there is the discussion reopened as a GItHub issue (#122) where everyone can raise their voice and which hopefully will lead to a consensus, or if not to a vote in TSC.

Reply:
I personally didn't need to document this discussion about the previous meeting in the minutes, but since it was included by Axel, I just wanted to include Telefónica's position and clarifications for the record. Because it was missing.